devito/propagator.py

```diff
@@ -165,6 +181,10 @@ def cfunction(self):
         return self._cfunction
 
     @property
+    def timings(self):
+        return self.profiler.as_dictionary
```

Comment:
Addressed in https://github.com/opesci/devito/pull/80/commits/8b21672a3e415e0008fc0fdbf9578cc3d0f4f4b8